### ISSUE TITLE
Early Earthquake Detection using BNGCNN

### ISSUE THREAD

**Issue by chippen001** [chippen001/Capstone-Resources#10]
### Title
Early Earthquake Detection Using Batch Normalization Graph Convolutional Neural Network (GNGCNN)

### URL
https://www.mdpi.com/2076-3417/12/15/7548

### Summary 
This article describes a GCNN model combining the strengths on CNNs and GNNs. The derived equations of the GCNN model are described. Batch normalization significantly reduced error and improved accuracy. The results showed reduced error rates across all measurements comparatively to the baseline model. 
 
### Key Points 

- Earthquakes are a major hazard to humans. Early detection can be developed through models with raw waveform data from seismic station sensors.
- Batch normalization applied to each mini-batch. Reduces epochs in training and even distribution of activation values. Improves model training and accuracy, fine-tunes parameters. 
- Deep learning models outperform traditional machine learning models.
- CNN extracts contextual features from input data. 
- GCNN good for processing multiple-station datasets.
- All error rates for longitude, latitude, depth, and magnitude had reduced error rate compared with baseline model.
- Future work: Explore other variants of models like GNN with attention layer mechanisms and transformers. Test models on multiple datasets with sparse and dense stations with and without station location information. Assess the damage done by an earthquake.

### Citation
Muhammad Atif Bilal, Yanju Ji, Yongzhi Wang, Muhammad Pervez Akhter, and Muhammad Yaqub. Early earthquake detection using batch normalization graph convolutional neural network (bngcnn). Applied Sciences, 12(15), 2022.
